### PR TITLE
Added login and fixed bugs

### PR DESCRIPTION
**Pull request by LIUBOMYR-HOLKO**

### CODE REVIEW

File: sports_hub/sports_hub/Controllers/AccountController.cs [hunk 1]
```diff
@@ -30,6 +33,43 @@ public IActionResult Login()
...
+                    {
+                        await Authenticate(model.Email);
+                    }
+
+                    return RedirectToAction("Index", "Home");
+                }
+                ModelState.AddModelError("", "Invalid email or password");
+            }
+            return View(model);
+        }
```
Comment (ZarechOleksii) [inline sports_hub/sports_hub/Controllers/AccountController.cs:56]: Порожній рядок тут

Comment (ZarechOleksii) [reply sports_hub/sports_hub/Controllers/AccountController.cs:56]: між методами

**Review by ZarechOleksii — APPROVED** [review]

File: sports_hub/sports_hub/Controllers/AccountController.cs [hunk 2]
```diff
@@ -30,6 +33,43 @@ public IActionResult Login()
...
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Login(LoginViewModel model)
+        {
+            if (ModelState.IsValid)
+            {
+                User user = await _userManager.FindByEmailAsync(model.Email);
+                if (user != null)
```
Comment (XashDM) [inline sports_hub/sports_hub/Controllers/AccountController.cs:43]: Empty line before if

File: sports_hub/sports_hub/Services/Implementations/EmailService.cs [hunk 3]
```diff
@@ -18,13 +20,16 @@ public async Task SendEmailAsync(string email, string subject, string message)
             emailMessage.From.Add(new MailboxAddress("Confirmation email", "sports.hub.project@gmail.com"));
             emailMessage.To.Add(new MailboxAddress("", email));
             emailMessage.Subject = subject;
-            //StreamReader streamReader = new StreamReader(@"C:\Users\ПК\source\repos\SportsHub\SportsHub\email_confirmation.html")/*(Directory.GetCurrentDirectory())*/;
-            //string html_register_template = streamReader.ReadToEnd();
+
+            //StreamReader streamReader = new StreamReader(@"C:\Users\ПК\source\repos\sport-hub\sports_hub\sports_hub\Views\Account\email_confirmation_cs.cshtml");
```
Comment (XashDM) [inline sports_hub/sports_hub/Services/Implementations/EmailService.cs:24]: Remove comments

File: sports_hub/sports_hub/Views/Account/email_confirmation_cs.cshtml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <div style="text-align: center;">
+            <h2 style="font-family: Open Sans; font-size: 24px; font-style: normal; font-weight: 600; line-height: 33px; letter-spacing: 0px; text-align: center;">Hello! You have successfully registered to Sports Hub</h2>
+            <h2 style="font-family: Open Sans; font-size: 24px; font-style: normal; font-weight: 600; line-height: 33px; letter-spacing: 0px; text-align: center;">
+                @(DateTime.Now.ToString("MMMM")+" " + DateTime.Now.Day.ToString() + ", " + DateTime.Now.Year.ToString())
+
+            </h2>
+
+        </div>
+        <div style=" background: #ffffff; height: 248px; width: 581px; border-radius: 0px; margin: auto; padding: 10px; text-align: center;">
+            <svg style="margin-top: 46px;" width="21" height="16" viewBox="0 0 21 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (XashDM) [inline sports_hub/sports_hub/Views/Account/email_confirmation_cs.cshtml:18]: Move to separate file

File: sports_hub/sports_hub/Views/Account/email_confirmation_cs.cshtml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Email confirmation";
+    Layout = "_EmailLayout";
+}
+<head>
+    <title>@ViewData["Title"]</title>
+</head>
+    <div style="background: #f9f9fb; border-radius: 0px; background-color: #f9f9fb">
+        <div style="text-align: center;">
+            <h2 style="font-family: Open Sans; font-size: 24px; font-style: normal; font-weight: 600; line-height: 33px; letter-spacing: 0px; text-align: center;">Hello! You have successfully registered to Sports Hub</h2>
```
Comment (XashDM) [inline sports_hub/sports_hub/Views/Account/email_confirmation_cs.cshtml:10]: Can we move this to separate style?

**Review by XashDM — APPROVED** [review]
Please fix all comments before merge

**Pull request merged**